tools/hangar/src/test_examples.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from "fs";
+import { join } from "path";
+
+export interface Example {
+  code: string;
+  filePath: string;
+  exampleNumber: number;
+}
+
+export function searchDirectoryForWingExamples(directory: string): Example[] {
```

Comment:
Duplicate code?

Reply:
Ah probably is, I moved things around from my original PR probably duplicated something with copy-pasta. Ill take a peek.